src/messages.c

```diff
@@ -1812,6 +1813,7 @@ void get_setting(coordinates_t loc, char *name, FILE* rsp)
 	GET_BOOL(remove_disabled_monitors)
 	GET_BOOL(remove_unplugged_monitors)
 	GET_BOOL(merge_overlapping_monitors)
+    GET_BOOL(record_history)
```

Comment:
You are using spaces for indentation here
```suggestion
	GET_BOOL(record_history)
```